gerritcheck/check.py

```diff
@@ -110,7 +110,12 @@ def cppcheck_on_files(files, commit):
 
     # Each line in the output is an issue
     review = {}
-    _, _, err = cppcheck_cmd.run(filter_files(files), retcode=None)
+
+    rc, out, err = cppcheck_cmd.run(filter_files(files), retcode=None)
+    if rc != 0:
```

Comment:
Cppcheck can returns non-zero exit code and valid messages at same time as well. If it crashes/fails during in progress.